### DISCUSSION TITLE
글씨 폰트가 너무 작습니다.

### DISCUSSION THREAD

**Discussion by DolDolDol** [metamong-friends/metamong-friends.github.io#7] [제보하기]
사용자명부터 화면에 뜨는 채팅 메세지까지 전체적으로 폰트가 작아서 잘 안보입니다.

**Comment by shlee303** [comment]
조치 예정입니다. 의견 감사합니다!

**Comment by shlee303** [comment]
폰트를 키워보았습니다! (5/7 18:30 반영 완료)
한번 사용해보시고 피드백 부탁드립니다 :)
